Cargo.toml

```
@@ -18,12 +18,12 @@ repository = "https://github.com/coreos/cargo-vendor-filterer"
 [dependencies]
 anyhow = "1.0"
 camino = "1.0"
-cargo-lock = "8"
-cargo_metadata = "0.11.3"
+cargo-lock = "10"
+cargo_metadata = "0.18.1"
 clap = { version = "4.1", features = ["derive", "wrap_help"] }
 flate2 = "1.0"
 hex = "0.4"
-toml = "0.5"
+toml = "0.8"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1.0.80"
```

```suggestion
serde_json = "1.0"
```
Cargo.lock bump indicates a change from "1.0.114" to "1.0.128", so listing .80 here does not add value in my opinion.